graphqlapi/pom.xml

```diff
@@ -189,6 +189,9 @@
         <configuration>
           <archive>
             <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+            <manifestEntries>
+              <x-Stargate-Health-Checkers>graphql</x-Stargate-Health-Checkers>
```

Comment:
Should probably be `graphqlapi` to match what you have for restapi.

Reply:
I was going by the service name (from the corresponding activator) in lower case.

Reply:
In the latest commit, the reuse of the activator name (lower case) is explicit (cf. `BaseActivator`).